gnosis/cowswap
# switcher added

# Summary

Fixes #578 
Recipient mode switcher was added to the Swap page in Setting menu. When turned on, the user should see the icon (was used one for example) that indicates that the mod is turned on, also the link `+ Add a send (optional)` should appear on the swap form just like it appears when Expert mode is on.

  # To Test

1. Open the page `Swap`
2. Open Settings
3. Turn on the Recipient Mode
4. Check out the indicator in the swap form header, checkout the link
